source/taup/install.rst

```diff
@@ -0,0 +1,47 @@
+安装
+====
+
+1. 确认 Java 运行环境已安装
+
+在终端中键入 ``java -version``，若显示版本信息，则表示 Java 运行环境已安装::
+
+    $ java -version
+    openjdk version "1.8.0_161"
+    OpenJDK Runtime Environment (build 1.8.0_161-b14)
+    OpenJDK 64-Bit Server VM (build 25.161-b14, mixed mode)
+
+否则需要先安装 Java 运行环境::
+
+    # 在 CentOS 7 上
+    $ sudo yum install java
+
+    # 在 Ubuntu 上
+    $ sudo apt update
+    $ sudo apt install default-jre default-jdk
```

Comment:
```suggestion
    # Ubuntu/Debian
    $ sudo apt update
    $ sudo apt install default-jdk
```

Reply:
How do you test different enviroments?

Reply:
I can't test it. It's based on my understanding of JDK and the search results from https://pkgs.org/, so they may work or not.